modules/http/src/test/java/org/mule/module/http/functional/requester/HttpRequestBuilderContentTypeTestCase.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.http.functional.requester;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.equalToIgnoringCase;
+import static org.hamcrest.Matchers.hasItem;
+
+import org.mule.api.MuleMessage;
+import org.mule.api.config.MuleProperties;
+import org.mule.tck.junit4.rule.DynamicPort;
+
+import java.io.IOException;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.server.Request;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class HttpRequestBuilderContentTypeTestCase extends AbstractHttpRequestTestCase
+{
+    @Rule
+    public DynamicPort httpListenerPort = new DynamicPort("httpListenerPort");
+
+    @Override
+    protected String getConfigFile()
+    {
+        return "http-request-builder-content-type-config.xml";
+    }
+
+    @Test
+    public void responseWithContentType() throws Exception
```

Comment:
test name is not representative. What you're really testing is that the request builder settings have precedence over the inbound properties. Provide a better test name. A brief javadoc explaining the case would also be nice ;)